shardingsphere-infra/pom.xml

```diff
@@ -40,5 +40,6 @@
         <module>shardingsphere-infra-context</module>
         <module>shardingsphere-infra-datetime</module>
         <module>shardingsphere-infra-optimize</module>
+        <module>shardingsphere-infra-optimizer</module>
```

Comment:
What's the difference between `shardingsphere-infra-optimize` and `shardingsphere-infra-optimizer`

Reply:
The module `shardingsphere-infra-optimize` can be regarded as an adaptor of calcite, and cannot optimize the join operator which can be pushdown to the sharding database. `shardingsphere-infra-optimizer` module does not depend on the `shardingsphere-infra-executor` which is different from `shardingsphere-infra-optimize` too. The `shardingsphere-infra-executor` will depend on the `shardingsphere-infra-optimizer` for the execution of the physical plan.